appserver/connectors/connectors-internal-api/src/main/java/com/sun/appserv/connectors/internal/api/ConnectorsClassLoaderUtil.java

```diff
@@ -118,7 +119,11 @@ private ConnectorClassFinder createRARClassLoader(final ClassLoader parent, Stri
                 if (processEnv.getProcessType().isEmbedded()) {
                     events.register(event -> {
                         if (event.is(EventTypes.PREPARE_SHUTDOWN)) {
-                            ccf.done();
+                            try {
+                                ccf.close();
+                            } catch(IOException ioe) {
+                                _logger.log(Level.WARNING, "error.closing.connector.classloader", ioe);
```

Comment:
I think this key doesn't exist anywhere ... you can just write an english message what failed, ie. Closing of the Connector Class Finder failed.